lib/MasterManifestGenerator.js

```diff
@@ -76,18 +47,25 @@ module.exports = function(entryId, hostname, port, applicationName, logger) {
                 }
                 else // Relative path
                 {
-                    return url.resolve(getMediaServerLiveUrl(), uri);
+                    return url.resolve(urlPath, uri);
                 }
             };
-
             return _.map(manifest.items.StreamItem, function(flavorObj){
+                //var urlPath = (newUrlFormat) ? newBehaviour().getMediaServerLiveUrl(flavors[flavorIndex++]) : wowzaBehaviour.getMediaServerLiveUrl();
+                var urlPath = manifestGenerator.getMediaServerLiveUrl(flavorIndex++);
```

Comment:
not sure what this is ?  the getMediaServerLiveUrl has different bheavour for the two type of the manifestGeneraotr?